home/ivp-planning/user-interface/the-info-panel.md

```diff
@@ -6,10 +6,10 @@ The info panel displays several information about the [selected object(s)](../ma
 
 ## The default fields:
 
-* _**ID:**_ the unique ID of the object which is generated when the machine is [placed ](../machines/first-steps-with-3d-object.md)in the [2D-](the-2d-panel.md) or [3D-panel](the-3d-panel.md); the ID is unchangeable and will persist until the object is [deleted](../machines/copy-and-delete-objects.md#delete-objects)
+* _**ID:**_ the unique ID of the object which is generated when the machine is [placed](../machines/first-steps-with-3d-object.md) in the [2D-panel](the-2d-panel.md) or [3D-panel](the-3d-panel.md); the ID is unchangeable and will persist until the object is [deleted](../machines/copy-and-delete-objects.md#delete-objects)
```

Comment:
```suggestion
* _**ID:**_ The unique ID of the object which is generated when the machine is [placed](../machines/first-steps-with-3d-object.md) in the [2D-](the-2d-panel.md) or [3D-panel](the-3d-panel.md). The ID is unchangeable and will persist until the object is [deleted](../machines/copy-and-delete-objects.md#delete-objects)
```

Capitalize the beginning of the sentences and simplify where possible (also applies to the points below).

Reply:
I have capitalized the beginning of the sentences, further simplification seems redundant though, as these are already quite simple and to the point.